src/views/Claiming/Claiming.tsx

```diff
@@ -184,11 +186,23 @@ export const Claiming: React.FC = () => {
 
   const queryFilter = filters?.[SEARCH_PARAMETER];
   const pageTitle = queryFilter ? intl.formatMessage({ id: 'ui-claims.document.title.search' }, { query: queryFilter }) : null;
+  const sortByOrg = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    if (sortingField !== CLAIMING_HIDDEN_LIST_COLUMNS.vendorId) {
+      changeSorting(e, { name: CLAIMING_HIDDEN_LIST_COLUMNS.vendorId });
+    }
+  };
 
   const renderActionMenu = ({ onToggle }: { onToggle: (e?: Event) => void }) => {
     return (
       <>
         <MenuSection>
+          <GroupByOrgActionMenuItem
+            onClick={(e) => {
+              onToggle(e as unknown as Event | undefined);
+              sortByOrg(e);
+            }}
+          />
+
```

Comment:
Please wrap this action item in `<IfPermission perm="orders.wrapper-pieces.collection.get">`